docs/CHANGELOG.asciidoc

```diff
@@ -208,9 +213,32 @@ This known issue impacts only {observability}, {stack-monitor-app}, and APM and
 *Workaround* +
 To work around this issue for {observability} and APM and {user-experience} rules, create them from the {observability} *Alerts* page.
 Refer to <<apm-alerts>> and {observability-guide}/create-alerts-rules.html[Create and manage {observability} rules].
+
+*Resolved* +
+This issue is resolved in 8.15.0.
 ====
 // end::known-issue-186969[]
 
+// tag::known-issue-189394[]
+.{webhook-cm} connector fails to send HTTP headers
+[%collapsible]
+====
+*Details* +
+If you configured the {webhook-cm} connector to send key-value pairs as headers, that information is not sent unles you have also enabled the basic authentication option for the connector.
+Refer to https://github.com/elastic/kibana/issues/189394[#189394].
+
+*Impact* +
+The impact of this issue will vary depending on the purpose of your headers.
```

Comment:
How about this?
```suggestion
The impact of this issue will vary depending on the purpose of your headers.
For example, if you added an `ApiKey` authorization header, you might receive a `401` authorization error since it's no longer sent by the connector.
```